src/main/proto/envoy/type/v3/http_status.proto

```
@@ -0,0 +1,199 @@
+syntax = "proto3";
+
+package envoy.type.v3;
+
+import "udpa/annotations/status.proto";
```

Buf (1.54.0)

import "udpa/annotations/status.proto": file does not exist

(COMPILE)